### ISSUE TITLE
Concept has an invalid reason, yet it is a standard concept

### ISSUE THREAD

**Issue by DinaSarsam** [OHDSI/Vocabulary-v5.0#65]
With the latest vocabulary 5 data downloaded, concept ID 8528 has an invalid reason of 'U' and Valid To Date of 01-APRIL-2015, yet is set as a standard concept (STANDARD_CONCEPT = 'S')

**Comment by ericaVoss** [comment]
@cgreich, in **v5.0 12-FEB-16** this still exists.

```{SQL}
SELECT *
FROM concept
WHERE STANDARD_CONCEPT = 'S'
AND INVALID_REASON IS NOT NULL
```

**Comment by ericaVoss** [comment]
This is fixed in **v5.0 11-MAR-16**
